apps/web/src/components/sidebar/SidebarNavigation/index.tsx

```diff
@@ -80,10 +75,19 @@ const Navigation = (): ReactElement => {
   }
 
   const handleNavigationClick = (href: string) => {
-    const eventInfo = customSidebarEvents[href]
-    if (eventInfo) {
-      trackEvent({ ...eventInfo.event, label: eventInfo.label })
-    }
+    const event = {
+      [AppRoutes.bridge]: BRIDGE_EVENTS.OPEN_BRIDGE,
+      [AppRoutes.swap]: SWAP_EVENTS.OPEN_SWAPS,
+      [AppRoutes.stake]: STAKE_EVENTS.OPEN_STAKE,
+      [AppRoutes.earn]: EARN_EVENTS.OPEN_EARN_PAGE,
+    }[href]
```

Comment:
This object is going to be recreated every time the handleNavigationClick function is executed. I doubt it would cause any performance issues, but still to be outside.

Plus if you were to modify the customSidebarEvents you would not have to touch those lines.